.devcontainer/docker-compose.yml

```diff
@@ -14,9 +14,10 @@ services:
     volumes:
       - "/tmp/.X11-unix:/tmp/.X11-unix"
       - "${HOME}/.vscode-server:/home/rcdt/.vscode-server"
-      - "../ros2_ws:/home/rcdt/ros2_ws"
+      - "../:/home/rcdt/rcdt_robotics/"
```

Comment:
Hier mounten we de hele repo aan de container. Is dit echt nodig/wat heb je nodig om pyflow aan de gang te krijgen?